mods/mc_worldmanager/hud.lua

```diff
@@ -8,21 +8,18 @@ local function createRealmInfoHudString(player)
     return string
 end
 
-function mc_worldManager.CreateRealmHud(player)
-    mc_worldManager.hud:add(player, "worldManager:currentRealm", {
-        hud_elem_type = "text",
-        position = { x = 1, y = 0 },
-        offset = { x = -16, y = 5 },
-        alignment = { x = "left", y = "down" },
-        text = createRealmInfoHudString(player),
-        color = 0xFFFFFF,
-    })
-end
-
 function mc_worldManager.UpdateRealmHud(player)
     if (not mc_worldManager.hud:exists(player, "worldManager:currentRealm")) then
-        return false
+        mc_worldManager.hud:add(player, "worldManager:currentRealm", {
+            hud_elem_type = "text",
+            position = { x = 1, y = 0 },
+            offset = { x = -16, y = 5 },
```

Comment:
The position of these stacked HUD elements conflict with the minimap. Can we move everything to the bottom left instead? Or offset it enough that it appears below or beside the minimap?

Reply:
I completely forgot that we had a minimap! This should be a pretty quick fix; I'll move it to the bottom left.